pkg/services/dummy/dummy.go

```diff
@@ -0,0 +1,215 @@
+/*
+ * Nuts auth
+ * Copyright (C) 2020. Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package dummy
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+
+	"github.com/nuts-foundation/nuts-auth/pkg/contract"
+	"github.com/nuts-foundation/nuts-auth/pkg/services"
+)
+
+// ContractFormat is the contract format type
+const ContractFormat = "dummy"
+
+// VerifiablePresentationType is the dummy verifiable presentation type
+const VerifiablePresentationType = "DummyVerifiablePresentation"
+
+// NoSignatureType is a VerifiablePresentation Proof type where no signature is given
+const NoSignatureType = "NoSignature"
+
+// SessionCreated represents the session state after creation
+const SessionCreated = "created"
```

Comment:
make a session status type

Reply:
added a more general SessionStatus type.

Reply:
down the road everything becomes a string. The type can't be changed for the irma signer/verifier. So it would be a Dummy only status type..... not worth all the casts